css/index.css

```diff
@@ -101,6 +120,9 @@ h1 {
   /*======= Create a selector for the address tag in the contact-section class to have the following styles:
      line-height: 2;
   */
+  .contact-section class{
+    line-height: 2;
+  }
```

Comment:
Your CSS is formatted correctly, but your selector is not. 
'.contact-section class' does not point to the address block.